app/views/content/life-as-a-teacher/pay-and-benefits/teacher-pay.md

```diff
@@ -92,7 +92,7 @@ Some schools also have a leading practitioner pay range for teachers with exempl
 What this will involve will depend on your school, but could include:
 
 * coaching and mentoring other teachers, including trainees and early career teachers
-* undertaking a national professional qualification (NPQ)
+* undertaking a national professional qualification (NPQ), such as the special educational needs (SENCO) NPQ to [specialise in teaching children with special educational needs](/life-as-a-teacher/pay-and-benefits/teacher-pay)
```

Comment:
I'm not particularly keen on this link - feels very shoehorned in and I think takes away from the wide range of NPQs available (https://www.gov.uk/guidance/national-professional-qualification-npq-courses) / distracts from what we are trying to inform people about on the pay page - let's discuss?

(There is also something odd going on in the spacing)
![image](https://github.com/user-attachments/assets/58ea3916-1753-48c7-b1e8-cc869cd51b01)

Reply:
Happy to remove, tbh I did shoehorn it in for SEO purposes, but there are probably more appropriate places to include a link, eg the upcoming secondary teacher page